### DISCUSSION TITLE
forge create error

### DISCUSSION THREAD

**Discussion by mehedihassan0** [Cyfrin/foundry-full-course-cu#559] [General]
as per video did, mine doesn't works  and throw different error 

```solidity 
 forge create SimpleStorage --rpc-url http://127.0.0.1:8545 

```
the errors 

```Error: 
error sending request for url (http://127.0.0.1:8545/): error trying to connect: tcp connect error: Connection refused (os error 111)

Context:
- Error #0: error trying to connect: tcp connect error: Connection refused (os error 111)
- Error #1: tcp connect error: Connection refused (os error 111)
- Error #2: Connection refused (os error 111)
```

**Comment by shikhar229169** [comment]
You need to start anvil in a new terminal. You can do it by running the below command in terminal:
```
anvil
```

**Reply by albin-george-kurian** [reply]
anvil should be running in order  to avoid this error..
1.open new terminal and type anvil
2.open another terminal and perform your task....
![Screenshot 2023-09-15 032750](https://github.com/Cyfrin/foundry-full-course-f23/assets/106237003/70f6a9be-202d-4a55-8d24-5bcabf8aa2a3)
